docs/howtos/avro_attrs_bridge.rst

```diff
@@ -58,3 +61,122 @@ If you want to go from attrs obj to avro dict and back:
    user_data_new_object = user_data_bridge.dict_to_attrs(user_data_dict)
    assert deserealize_user_data == user_data_new_object
 
+
+How to extend AvroAttrsBridge class
+-----------------------------------
+
+At default, attrs.asdict only supports basics types for conversion to dict (Basically, only things you could json.dump). To allow AvroAttrsBridge to work with non-primitive types, a function will be passed to  value_serializer arg in attrs.asdict. The value_serializer function needs to be able to handle any non-primitive types used in an events attrs class.
+
+To make is easier to developers, an extensions interface has been implemented into AvroAttrsBridge.
+To allow AvroAttrsBridge to work with these classes, you can pass in a dict to the extensions keyword to AvroAttrsBridge. The extensions keyword expects a dict with following format: {<type of non-primitive>: <AvroAttrsBridgeExtention subclass for non-primitive>}
```

Comment:
I don't understand "pass in a dict to the extensions keyword." How does one pass a dict to a keyword? Did you mean attribute?